src/app/app.component.ts

```diff
@@ -1,9 +1,16 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { PrimeNGConfig } from 'primeng/api';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
 })
-export default class AppComponent {
-  title = 'sample-angular-app';
+export default class AppComponent implements OnInit {
+  constructor(private primengConfig: PrimeNGConfig) {
+    console.log('constructor logic');
```

Comment:
console.logs should not be present as per eslint

Reply:
Add disbale rule for line for empty constructor instead of console log

Reply:
I have added inline disable comments and removed console logs in constructors.